src/main/java/frc/robot/subsystems/Intake/Intake.java

```diff
@@ -0,0 +1,52 @@
+package frc.robot.subsystems.Intake;
+
+
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.StatusSignal;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Direction;
+
+
+
+
+abstract public class Intake extends SubsystemBase {
+    protected final TalonFX intakeMotor;
+
+    public Intake() {
+        intakeMotor = new TalonFX( 1);
+    }
+
+    public Command runIntake() {
+        return runOnce(() -> intakeMotor.set(Constants.Intake.INTAKE_SPEED));
+    }
+
+    public Command reverseIntake() {
+        return runOnce(() -> intakeMotor.set(-1 * Constants.Intake.INTAKE_SPEED));
+    }
+
+    public Command stopIntake() {
+        return runOnce(() -> intakeMotor.set(0.0));
+    }
+
+    public Command lowerIntake() { return Commands.none(); }
+
+    public Command raiseIntake() { return Commands.none(); }
+
+
+    public Command changeState(Constants.IntakeState newState) {
```

Comment:
this should basically be the only public method in intake, the rest should be private unless they are intended to be used for debugging/manual control